prac_05/word_occurences.py

```diff
@@ -0,0 +1,19 @@
+"""Program to count occurrences of words in a string and return list of words with their count."""
+
+
+def main():
+    """Program to count occurrences of words in a string."""
+    sentence = input("Enter sentence: ").split()
```

Comment:
`sentence` is a good name for a sentence. A list of words would be called `words`.